.travis.yml

```diff
@@ -10,6 +10,12 @@ notifications:
 
 # Install stuff
 before_install:
+  - hostname
+  - cat /etc/hosts # optionally check the content *before*
+  - sudo hostname "$(hostname | cut -c1-63)"
+  - sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts | sudo tee /etc/hosts
```

Comment:
You need to update line 8 in `install_sge.sh` to account for this change.  It also modifies `/etc/hosts` and it won't be able to find the pattern it's looking for until you update that.